app/xlist/scripts/HomeController.js

```diff
@@ -124,9 +124,28 @@ angular
             });
           });
         }).then(function() {
-          $scope.pairs = _.sortBy(newPairs, function(pair) {
-            return pair.geoList.name;
-          });
+          if (newPairs.length === 0) {
+            var newGeoPoint = new ParseObject(new GeoList(), GeoList.fields);
+            newGeoPoint.name = 'To-Do List';
+            newGeoPoint.location = new Parse.GeoPoint({
+              latitude: 0.0,
```

Comment:
Let's tackle it in multiple lists. Most of this code will be scrapped anyway once we get that up and running.